src/element/shape.js

```diff
@@ -599,8 +599,8 @@ export class ShapeElement extends Element {
       const paths = shapePath.path.paths;
       if (frame < paths[0].startFrame) {
         this.drawPath(paths[0].fromPath);
-        if (index !== 0 && this.graphicsData.length > 1) {
-          this.addHole();
+        if (index !== 0) {
```

Comment:
Its removed graphicsData because is not supported by the class PIXI.Graphics in V5, but in this case I need to deep more inside to find a other property to get the last index for _this.GraphicsData.length > 1_  😓 😓 😓